example/lib/main.dart

```diff
@@ -52,17 +52,17 @@ class _MyAppState extends State<MyApp> {
   }
 
   Future<void> getPermissionStatus() async {
-    bool permissionStatus = false;
+    bool? permissionStatus = false;
     try {
-      permissionStatus = await PermissionHandler.permissionsGranted;
+      permissionStatus = await (PermissionHandler.permissionsGranted as FutureOr<bool>);
```

Comment:
Could you remove `as FutureOr<bool>` from this line? It causes the PermissionHandler to not work as intended. The console logs the following line while launching the app from your pull request:
```
type 'Future<bool?>' is not a subtype of type 'FutureOr<bool>' in type cast
```

Reply:
Removed, thanks!